src/plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -149,7 +149,11 @@ const handleCourierRequest = async ({
   request.stats(getRequestInspectorStats(requestSearchSource));
 
   try {
-    const response = await requestSearchSource.fetch({ abortSignal });
+    const response = await requestSearchSource.fetch({
```

Comment:
lets remove this for now, we don't need it for discover. I think we'll need to pass the session id in to the expressions rather than accessing it directly like this.